### PR TITLE
fix a mission-critical bug in the framework software patch

### PR DESCRIPTION
**Pull request by not-so-smart**
this really important and large PR with lots of changes and substantial contributions adds heaps of functionality to the robot as well as marking me as a contributor on this repository and granting me a very small ego boost

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Kyoyobot
+this bot is really cool and I contributed to it
```
Comment (96-LB) [inline README.md:2]:
```suggestion
```

**Comment by 96-LB** [comment]
Looks great! I just have one suggestion; see the review for more information.

File: README.md (hunk 1 above)
Comment (IdleSicko) [reply README.md:2]: nice

**Comment by Sanketh7** [comment]
Looks like a great change, but we will have to get approval from legal and upper management before we can push this change.

**Comment by jason-liau** [comment]
LGTM!

**Pull request merged**